morphic.js

```diff
@@ -7860,6 +7909,41 @@ StringMorph.prototype.endOfLine = function () {
     return this.text.length;
 };
 
+StringMorph.prototype.previousWordFrom = function (aSlot) {
+    // answer the slot (index) slots indicating the position of the
+    // previous word to the left of aSlot
+    var index = aSlot - 1;
+    
+    // while the current character is non-word one, we skip it, so that
+    // if we are in the middle of a non-alphanumeric sequence, we'll get
+    // right to the beginning of the previous word
+    while (index > 0 && !isWordChar(this.text[index])) {
+        index --;
```

Comment:
another little quib: Some of the static code analysis tools I use dislike shifting integers and complain about this. Can we change it to something a little bit more explicit, like ```index -= 1; ``` ? Thanks! (And sorry for the hassle)